### PR TITLE
Update doxygen comments of the Cooperative Group API

### PR DESCRIPTION
**Pull request by neon60**

### CODE REVIEW

**Review by parbenc — DISMISSED** [review]
LGTM

**Comment by matyas-streamhpc** [comment]
lgtm.

**Review by matyas-streamhpc — DISMISSED** [review]

**Comment by neon60** [comment]
@chrispaquot rebased the brand. Thanks for the notification.

File: hipamd/include/hip/amd_detail/amd_hip_cooperative_groups.h [hunk 1]
```diff
@@ -550,7 +550,7 @@ class coalesced_group : public thread_group {
 
   /** \brief Ballot function on group level.
    *
-   *  \details Returns a bit mask with Nth bit is set to one, if Nth thread
+   *  \details Returns a bit mask with Nth bit is set to one if Nth thread
```
Comment (randyh62) [inline hipamd/include/hip/amd_detail/amd_hip_cooperative_groups.h]: Returns a bit mask with the Nth bit set to one if the specified predicate evaluates as true on the Nth thread.

File: hipamd/include/hip/amd_detail/amd_hip_cooperative_groups.h [hunk 2]
```diff
@@ -603,7 +603,8 @@ class coalesced_group : public thread_group {
    *           participating threads have the same value in ``value``.
    *
    *  \param value [in] The value of the current thread in group.
```
Comment (randyh62) [inline hipamd/include/hip/amd_detail/amd_hip_cooperative_groups.h]: The value to examine on the current thread in group.

File: hipamd/include/hip/amd_detail/amd_hip_cooperative_groups.h [hunk 3]
```diff
@@ -603,7 +603,8 @@ class coalesced_group : public thread_group {
...
-   *  \param pred [out] The result of all participating threads have the same ``value``.
+   *  \param pred [out] The predicate is set true if all participating threads
```
Comment (randyh62) [inline hipamd/include/hip/amd_detail/amd_hip_cooperative_groups.h]: The predicate is set to true if all participating threads in the thread group have the specified value.

File: hipamd/include/hip/amd_detail/amd_hip_cooperative_groups.h [hunk 4]
```diff
@@ -1092,7 +1093,7 @@ class thread_block_tile : public impl::thread_block_tile_internal<size, ParentCG
 
   /** \brief Ballot function on group level.
    *
-   *  \details Returns a bit mask with Nth bit is set to one, if Nth thread
+   *  \details Returns a bit mask with Nth bit is set to one if Nth thread
```
Comment (randyh62) [inline hipamd/include/hip/amd_detail/amd_hip_cooperative_groups.h]: Returns a bit mask with the Nth bit set to one if the Nth thread predicate evaluates true.

**Review by randyh62 — DISMISSED** [review]

File: hipamd/include/hip/amd_detail/amd_hip_cooperative_groups.h (hunk 3 above)
Comment (neon60) [reply hipamd/include/hip/amd_detail/amd_hip_cooperative_groups.h]: The predicate is set to true if all participating threads in the thread group have the same value.

**Pull request merged**